pg_search/src/postgres/storage/linked_items.rs

```diff
@@ -209,6 +209,7 @@ impl<T: From<PgItem> + Into<PgItem> + Debug + Clone + MVCCEntry> LinkedItemList<
     ///
     pub unsafe fn retain(
         &mut self,
+        when_recyclable: pg_sys::TransactionId,
         mut f: impl FnMut(&mut BufferManager, T) -> RetainItem<T>,
     ) -> Vec<T> {
```

Comment:
Not the time to tackle it now, but it feels like `LinkedItemList` could use a shift in its API for a universe where lists only grow... and when they shrink, it's only because they are inside an `atomically` block and not yet visible.